packages/odyssey-react-mui/src/labs/GroupPicker.tsx

```diff
@@ -0,0 +1,247 @@
+      return (
+        <li {...props} key={option.id}>
+          <Box
+            sx={{
+              display: "flex",
+              flexDirection: "row",
+              alignItems: "top",
+            }}
+          >
+            <Box sx={{ paddingRight: odysseyDesignTokens.Spacing2 }}>
+              <MuiAvatar
+                sx={{
+                  backgroundColor: "transparent",
+                  width: 24,
+                  height: 24,
+                }}
+                src={option.logo}
+              >
+                &nbsp;
+              </MuiAvatar>
+            </Box>
+            <Box>
+              {option.name}
+              <Subordinate>{option.description}</Subordinate>
+              <Box
+                sx={{
+                  display: "flex",
+                  flexDirection: "row",
+                  paddingTop: odysseyDesignTokens.Spacing1,
+                }}
+              >
+                {option.usersCount !== undefined && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      flexDirection: "row",
+                      paddingRight: odysseyDesignTokens.Spacing4,
+                    }}
+                  >
+                    <UserIcon />
+                    {option.usersCount}
+                  </Box>
+                )}
+                {option.appsCount !== undefined && (
+                  <Box
+                    sx={{
+                      display: "flex",
+                      flexDirection: "row",
+                      paddingRight: odysseyDesignTokens.Spacing4,
+                    }}
+                  >
+                    <GridIcon />
+                    {option.appsCount}
+                  </Box>
+                )}
+                {option.groupPushMappingsCount !== undefined && (
+                  <Box sx={{ display: "flex", flexDirection: "row" }}>
+                    <GroupIcon />
+                    {option.groupPushMappingsCount}
+                  </Box>
+                )}
+              </Box>
+            </Box>
+          </Box>
+        </li>
+      );
+    },
+    [odysseyDesignTokens]
+  );
+
+  const renderTags = useCallback(
+    (values: optionType[], getTagProps: AutocompleteGetTagProps) =>
+      values.map((option: optionType, index: number) => {
+        const { onDelete } = getTagProps({ index });
+
+        return (
+          <div key={option.id}>
+            <MuiChip
+              sx={{ margin: odysseyDesignTokens.Spacing1 }}
+              size="small"
+              avatar={
+                <MuiAvatar
+                  sx={{
+                    left: `-${odysseyDesignTokens.Spacing2}`,
+                    backgroundColor: "transparent",
+                  }}
+                  src={option.logo}
+                >
+                  &nbsp;
+                </MuiAvatar>
+              }
+              label={option.name}
+              onDelete={onDelete}
+            />
```

Comment:
If we can use the existing Odyssey `Tag` component, I'd prefer that over redoing `MuiChip` with a negative `left` style on it.

I know that `Tag` doesn't accept images (only icons), so we'll wanna figure that out.

Reply:
The type could take an `icon` or an `imageUrl` prop.

Reply:
I keep the `Chip` until `Tag` can support image.